lib/about_yml/schema.json

```diff
@@ -6,127 +6,127 @@
-            "items": {"type": "string"},
-            "uniqueItems": true
+            "description": "MWhat is your project's current status?",
+            "enum": ["discovery", "alpha", "beta", "live"]
         },
         "testable": {
             "type": "boolean",
             "default": false,
             "description": "Should this repo have automated tests? If so, set to `true`."
         },
-        "team": {
-            "type": "array",
-            "description": "Team members contributing to the project",
-            "items": {
-                "$ref": "#/definitions/person"
-            },
-            "uniqueItems": true
+        "licenses": {
+            "type": "object",
+            "description": "What are the licences that apply to the project and/or its components?",
+            "patternProperties": {
+                ".*": {"$ref": "#/definitions/license"}
+            }
         },
         "partners": {
             "type": "array",
-            "description": "Partners for whom the project is developed",
+            "description": "Who is the parter for your project? (Use the full name of the partner documented here: https://github.com/18F/dashboard/blob/staging/_data/partners.yml)",
             "items": {"type": "string"},
             "uniqueItems": true
         },
-        "users": {
+        "contact": {
             "type": "array",
-            "description": "Organizations or individuals who have adopted the project for their own use",
+            "description": "The main point of contact(s) for your project, and a `mailto:` link for each contact.",
             "items": {
-                "$ref": "#/definitions/user"
+                "$ref": "#/definitions/contact"
             },
             "uniqueItems": true
         },
-        "milestones": {
+        "team": {
             "type": "array",
-            "description": "Brief descriptions of significant project developments",
-            "items": {"type": "string"},
+            "description": "Who are the team members on your project?",
+            "items": {
+                "$ref": "#/definitions/person"
+            },
             "uniqueItems": true
         },
-        "stack": {
-            "type": "array",
-            "description": "Technologies used to build the project",
-            "items": {"type": "string"},
-            "uniqueItems": true
+        "type": {
+            "type": "string",
+            "description": "What kind of content is contained in the project repository?",
+            "enum": ["app", "docs", "policy"]
         },
-        "impact": {
+        "parent": {
             "type": "string",
-            "description": "Brief description of the project's outcomes"
+            "description": "Name of the main project repo if this is a sub-repo; name of the working group/guild repo if this is a working group/guild subproject"
```

Comment:
I'd be down with replacing "working group/guild" with "grouplet" here. But I'm not pushing hard for this.